daml/splice-amulet-test/daml/Splice/Scripts/TestExpiry.daml

```diff
@@ -1,116 +0,0 @@
--- Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
I noticed a bunch of unit tests that were not below `Splice.Scripts.UnitTests` when I was looking for a good blueprint for the test to write. Decided to move them as a drive-by contribution. No functional change in the deleted code. Just merged in the unit test scripts for the module they test.